.eslintrc.json

```diff
@@ -76,8 +75,7 @@
     "jsdoc/require-param-type": "off",
     "jsdoc/require-returns-check": "error",
     "jsdoc/require-returns-description": "error",
-    "jsdoc/require-returns-type": "off",
-    "tsdoc/syntax": "error"
+    "jsdoc/require-returns-type": "off"
```

Comment:
Removing tsdoc because it is not compatible with how we have elected to define object properties (ie. @param options.someProperty is valid jsdoc but invalid tsdoc) and tsdoc currently has no straightforward mechanism for toggling applied rules.